FileAPI/FileReader/test_notreadableerrors-manual.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>FileReader NotReadableError Test</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<link rel="help" href="http://www.w3.org/TR/FileAPI/#dfn-error-codes">
```

Comment:
Please reference the ED version of the spec: https://w3c.github.io/FileAPI/#dfn-error-codes

Reply:
Updated.